Library/Formula/xulrunner.rb

```diff
@@ -25,7 +23,7 @@ class Xulrunner < Formula
   homepage "https://developer.mozilla.org/docs/XULRunner"
 
   stable do
-    url "https://ftp.mozilla.org/pub/mozilla.org/xulrunner/releases/latest/source/xulrunner-33.0.source.tar.bz2"
+    url "https://ftp.mozilla.org/pub/mozilla.org/xulrunner/releases/33.0/source/xulrunner-33.0.source.tar.bz2"
```

Comment:
Could we add a comment to not use `latest` in the URL so we don't accidentally break this again in future? Thanks!